### PR TITLE
[DOCFIX] Add docs for dynamical log level change

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: docs/_data/table/en/Developer-Tips.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ loadufs:
   Loads files under <code>UfsPath</code> to the given <code>AlluxioPath</code>.
   <code>ExcludePathPrefixes</code> can be a set of prefixes which are separated by ';'.
   The paths with a prefix in <code>ExcludePathPrefixes</code> will not be loaded.
+logLevel:
+  Dynamically changes the log level of a class on the running Alluxio server.
+  Try alluxio journalCrashTest -help to see the meanings of each argument in detail
```
Comment (aaudiber) [inline docs/_data/table/en/Developer-Tips.yml]: added by mistake?

File: docs/en/Logging-Conventions.md [hunk 2]
```diff
@@ -120,3 +120,27 @@ log4j.appender.MASTER_LOGGER_SOCKET.layout.ConversionPattern=%d{ISO8601} %-5p %c
...
+```bash
+alluxio logLevel --loggerName=foo.Bar --target=master,192.168.100.100:19999 --level=DEBUG
+```
+
+
+And the following command gets all workers' log level on class `foo.Bar`
+```bash
+alluxio logLevel --loggerName=foo.Bar --target=workers
+```
+
```
Comment (aaudiber) [inline docs/en/Logging-Conventions.md]: extra newlines at end

File: docs/en/Logging-Conventions.md [hunk 3]
```diff
@@ -120,3 +120,27 @@ log4j.appender.MASTER_LOGGER_SOCKET.layout.ConversionPattern=%d{ISO8601} %-5p %c
...
+indicates the logger's name, and `target` lists the Alluxio masters or workers to set. If parameter `level` is provided the command
+changes the logger level, otherwise it gets and displays the current logger level.
+
+For example, this command sets the class `foo.Bar`'s logger level to DEBUG on master as well as a worker at `192.168.100.100:19999`.
+
+```bash
+alluxio logLevel --loggerName=foo.Bar --target=master,192.168.100.100:19999 --level=DEBUG
+```
+
+
```
Comment (aaudiber) [inline docs/en/Logging-Conventions.md]: extra newline

File: docs/en/Logging-Conventions.md [hunk 4]
```diff
@@ -120,3 +120,27 @@ log4j.appender.MASTER_LOGGER_SOCKET.layout.ConversionPattern=%d{ISO8601} %-5p %c
...
+### Dynamically change the log level when Alluxio server is running
+
+Alluxio shell comes with a `logLevel` command that allows you to get or change the log level of a particular class on specific
+instances.
+
+The synax is `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`, where the `logName`
+indicates the logger's name, and `target` lists the Alluxio masters or workers to set. If parameter `level` is provided the command
+changes the logger level, otherwise it gets and displays the current logger level.
+
+For example, this command sets the class `foo.Bar`'s logger level to DEBUG on master as well as a worker at `192.168.100.100:19999`.
```
Comment (aaudiber) [inline docs/en/Logging-Conventions.md]: worker uses 30000 for its web port by default

**Comment by yupeng9** [comment]
@aaudiber comments addressed. PTAL.

File: docs/_data/table/en/Developer-Tips.yml [hunk 5]
```diff
@@ -13,6 +13,9 @@ loadufs:
   Loads files under <code>UfsPath</code> to the given <code>AlluxioPath</code>.
   <code>ExcludePathPrefixes</code> can be a set of prefixes which are separated by ';'.
   The paths with a prefix in <code>ExcludePathPrefixes</code> will not be loaded.
+logLevel:
+  Dynamically changes the log level of a class on the running Alluxio server.
+  Try alluxio logLevel -help to see the meanings of each argument in details.
```
Comment (aaudiber) [inline docs/_data/table/en/Developer-Tips.yml]: nit: meanings -> meaning, details -> detail

**Pull request merged**